src/Uno.UI/Microsoft/UI/Xaml/Controls/RadioMenuFlyoutItem/RadioMenuFlyoutItem.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	// UNO Docs:
+	// In WinUI 2.6. RadioMenuFlyoutItem derives publically from MenuFlyoutItem, but secretly from ToggleMenuFlyoutItem.
+	// Since we can't do that in C#, the important functionality are in ToggleMenuFlyoutItem, we derive from it.
+	public partial class RadioMenuFlyoutItem : ToggleMenuFlyoutItem
```

Comment:
Unfortunately the inheritance hierarchy doesn't match WinUI, and it's going to be somewhat hard to fix that.

Probably the multiple inheritance may be removed in WinUI 3.0. When/if that happen, we can fix that (will be a small breaking change)